### PR TITLE
Atom Package Manager (apm) Bash-Completion

### PR DESCRIPTION
**Pull request by vigo**
Bash Completion for [Atom](http://atom.io) Package Manager

### CODE REVIEW

File: apm-bash-completion.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class ApmBashCompletion < Formula
+  desc "Bash completion for Atom Package Manager (apm)"
```
Comment (bfontaine) [inline apm-bash-completion.rb]: Please shorten this description. “homebrew/completions/apm-bash-completion: <desc>” should fit under 80 characters.

Comment (vigo) [inline apm-bash-completion.rb]: `Bash completion for Atom Package Manager (apm)` 47 chars? Can you help me out?

Comment (bfontaine) [inline apm-bash-completion.rb]:
```
$ echo -n "homebrew/completions/apm-bash-completion: Bash completion for Atom Package Manager (apm)"|wc -c
      88
```

You could use e.g. `Bash completion for apm`.

Comment (vigo) [inline apm-bash-completion.rb]: aha... ok, I've got it... Thanks

**Comment by vigo** [comment]
what can I do about these?

- GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)
- GitHub repository too new (<30 days old)

:)

**Comment by bfontaine** [comment]
This tap is more permissive than the core so it’s ok here. Thank you for your contribution; we appreciate it :wink:

**Pull request closed without merging**